app/index.ts

```diff
@@ -25,7 +25,7 @@ messaging.peerSocket.onopen = () => {
   changeScreen(new ZoneGroupScreen(changeScreen));
 };
 
-messaging.peerSocket.onmessage = (evt: any): void => {
+messaging.peerSocket.onmessage = (evt): void => {
```

Comment:
Not `any` anymore: it's [`MessageEvent`](https://github.com/SergioMorchon/fitbit-sdk-types/blob/master/shared/messaging.d.ts#L35).